tealish/nodes.py

```diff
@@ -1374,28 +1359,33 @@ def _tealish(self) -> str:
 
 class Func(InlineStatement):
     possible_child_nodes = [InlineStatement]
-    pattern = r"func (?P<name>[a-zA-Z_0-9]+)\((?P<args>.*)\)(?P<returns>.*):$"
+    pattern = r"func (?P<name>[a-zA-Z_0-9]+)\((?P<args>.*)\)(?P<return_type>.*):$"
     name: str
     args: ArgsList
+
+    return_type: str
```

Comment:
this "fixes" the issue of the str vs List[str], im not sure its the _right_ way but it is _a_ way, lmk what you think